test/spark/docker-compose.yaml

```diff
@@ -0,0 +1,88 @@
+version: '3.9'
+services:
+  postgres:
+    image: "postgres:11"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: lakefs
+      POSTGRES_PASSWORD: lakefs
+  lakefs:
+    image: "${REPO}/lakefs:${TAG}"
+    ports:
+      - "8000:8000"
+    networks:
+      default:
+        ipv4_address: 10.5.0.55
+    depends_on:
+      - "postgres"
+    volumes:
+      - ./lakectl-tester.yaml:/home/lakefs/.lakectl.yaml
+      - .:/local
```

Comment:
Not sure why we expose CWD to the container.

Reply:
need access to the dataset in order to upload it part of the test